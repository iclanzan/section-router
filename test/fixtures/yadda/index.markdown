# This is more interesting

The generated file should be `/yadda/index.html`.
