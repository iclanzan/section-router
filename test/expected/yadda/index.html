<p>The generated file should be <code>/yadda/index.html</code>.</p>